client/state/notices/actions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import {
+	NEW_NOTICE,
+	REMOVE_NOTICE
+} from 'state/action-types';
+
+import { uniqueId } from 'lodash';
```

Comment:
Will any of this run on the server? Because I believe `uniqueId` will then generate unique IDs across all sessions... so I think we could potentially run into memory issues?